excellent/functions/functions.go

```diff
@@ -1419,10 +1528,59 @@ func FormatDateTime(env utils.Environment, args ...types.XValue) types.XValue {
 		date = types.NewXDateTime(date.Native().In(location))
 	}
 
-	// return the formatted date
 	return types.NewXText(date.Native().Format(goFormat))
 }
 
+// FormatTime formats `time` as text according to the given `format`.
```

Comment:
Docs should mention that if format is not specified then the default format is used.